dotnet-roslyn/analyzers/Audacia.CodeAnalysis.Analyzers/src/Audacia.CodeAnalysis.Analyzers/Audacia.CodeAnalysis.Analyzers.csproj

```diff
@@ -17,7 +17,8 @@
         <PackageTags>Audacia.CodeAnalysis.Analyzers, analyzers</PackageTags>
         <NoPackageAnalysis>true</NoPackageAnalysis>
         <PackageIcon>package-icon.png</PackageIcon>
-        <Version>1.9.0</Version>
+        <Version>1.9.1</Version>
+        <GeneratePackageOnBuild>False</GeneratePackageOnBuild>
```

Comment:
question: is this needed?

Reply:
I tried publishing a local package for testing and then turned it off and that might have added this in. I'll get that removed,